content/v0.80.0.md

```diff
@@ -0,0 +1,478 @@
+- #2000: Mouse mapper and configuration mechanism
+- #2037: Contribute a script to convert from YouTube chapters to CD-DA
+- #1996: Add direct support for Windows 3.1 seamless mouse driver by javispedro
+- #2137: Add a subset of OpenGL shaders from Tyrells repo
+- #2127: Customise the looks of the macOS DMG distribution disk image
+- #2129: Bump microsoft/setup-msbuild from 1.2 to 1.3
+- #2124: Drop architecture spec from setup-msbuild GitHub action
+- #2123: Bump microsoft/setup-msbuild from 1.1 to 1.2
+- #2101: Debloat the binary: use more heap storage and retire the software scalar
+- #2104: Cleanup overcomplicated data conversions in Unicode support
+- #2108: Check file-descriptor dependency for Enet, slirp, and ManyMouse
+- #2103: Generate linker map file by default
+- #2099: Add more cleanup on audio device tear-down
+- #2097: Update wraps and PVS Studio
+- #2096: Cleanup a handful of build issues for clang 10.x and 11.x
+- #2092: Sync reSIDfp with upstream
+- #2084: Cleanup issues in ReelMagic flagged by Coverity
+- #2085: Look for libXInput dependency
+- #2069: Update Tracy to v0.9
+- #2075: Add libraries to support TrueType font rendering
+- #2076: Compiler warning fix in ManyMouse library
+- #2071: Sync reSIDfp with upstream
+- #2068: Fix sentence starting with lower case in MOUNT command help
+- #2064: Cleanup string IDs of DOS programs
+- #2060: Minor mouse enhancements
+- #2040: Fix a couple CWE issues in ManyMouse
+- #2036: Add release drafter support files
+- #2031: Update CI against latest deprecations and OS versions
+- #2028: Bump actions/download-artifact from 2 to 3
+- #2027: Bump fountainhead/action-wait-for-check from 1.0.0 to 1.1.0
+- #2026: Bump actions/upload-artifact from 2 to 3
+- #2025: Bump actions/cache from 2 to 3.0.11
+- #2024: Update the build system wraps and automate CI action maintenance
+- #1992: Improve detection of atomics, testing of speexDSP, and cleanup README
+- #2130: simplifies file search in virtual drive
+- #2116: Use a `sdd::unique_ptr` to manage the video memory
+- #2090: Refactor the ReelMagic audio FIFO class
+- #2086: Refactor ReelMagic player with per-stream handle registration
+- #2045: Split contributed CD-DA scripts into fetch vs. cue functions
+- #2060: Minor mouse enhancements
+- #2117: Use shared pointers to manage the virtual files
+
+
+</details>
+
+##  6. <a name='Bugandregressionfixes'></a>Bug and regression fixes
+
+- The mixer's effects (reverb, chorus, crossfeed) can now be turned on and off
+  at runtime via the mixer's CLI.
+
+- FluidSynth's specific reverb and chorus configuration settings are now applied
+  and will replacing the default values that were previously always applied.
+
+- The modem's call now has a warm-up period where it will discard data. This
+  fixes a bug noted in Duke Nukem 3D where it would fail it's initial handshake
+  in the first try.
+
+<details>
+  <summary>Log of 0.80's pull requests involving bug and regression fixes:</summary>
```

Comment:
Needs blank link after the `<summary>`, otherwise won't be displayed as bulleted list.

Reply:
Thanks; I've done this - 

![2022-12-20_17-22](https://user-images.githubusercontent.com/1557255/208798345-ca5d0360-c956-4dc0-878a-b3f13561c997.png)